blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
in general, I strongly disagree that we should add this no script block. It would imply we have no server side rendering solution available when there is available. Moreover, we plan to add the fastboot addon in the default blueprint when creating apps. So this message is surely going to be confusing.

Also there is no way to check if fastboot is used since it will be a runtime thing rather than static. Fastboot could remove the noscript block before sending the index.html but it seems wrong to me.